[core] Batch small changes (#27435)
mui-org/material-ui: docs/src/pages/guides/api/api.md

### Property naming

The name of a boolean prop should be chosen based on the **default value**. This choice allows:

- the shorthand notation. For example, the `disabled` attribute on an input element, if supplied, defaults to `true`:

  ```jsx
  <Input enabled={false} /> ❌
  <Input disabled /> ✅
  ```

> - developers knowing what's the default value only reading the name of the boolean prop. It's always `false`.

Reviewer: ```suggestion
- developers to know what the default value is from the name of the boolean prop. It's always the opposite.
```
